app/views/admin/loans/content/_form.html.slim

```diff
@@ -10,18 +10,29 @@
 
   = f.error_notification
 
-  - if @loan.division && division_policy(@loan.division).show?
-    = f.input :division
-      .view-element
-        - if @loan.division
+  = f.input :division
+    .view-element
+       - if @loan.division && division_policy(@loan.division).show?
           = link_to @loan.division.name, admin_division_path(@loan.division_id)
-      = f.input_field :division_id, collection: @division_choices, include_blank: false
+       - else
+        = @loan.division.try(:name)
+    .form-element
+      - if !@loan.division || division_policy(@loan.division).show?
+        = f.input_field :division_id, collection: @division_choices, include_blank: false
+      - else
+        = @loan.division.try(:name)
```

Comment:
I was thinking to not make the display code dependent upon that assumption in case the 'may_edit?' logic changes in the future.  I've removed the try, but added comments warning of the dependency as the with the refactoring out to helper method and partial as further separated the logic and dependency.